src/main/java/de/craften/ui/swingmaterial/MaterialFrame.java

```diff
@@ -0,0 +1,79 @@
+package de.craften.ui.swingmaterial;
+
+import java.awt.Color;
+import java.awt.Container;
+import java.awt.event.WindowEvent;
+import javax.swing.JPanel;
+
+/**
+ * An extension of {@link MaterialWindow} with a classic title bar, control
+ * buttons and the ability to drag, minimize and maximize the window as the user
+ * wishes. Specially recommended when you want your application to behave like
+ * common desktop apps. If you are looking for a behavior more like mobile apps,
+ * try {@code MaterialAppFrame}.

```

Comment:
Oh wait, what is *that* doing there? I though I had removed it in the 3rd commit.

I mean, there's no such a thing as a `MaterialAppFrame` *yet*. We would need a proper Navbar and Sidebar first.